### PR TITLE
+ jsDelivr link

### PR DESCRIPTION
**Pull request by tomByrer**
BTW, `script type="text/javascript"`  isn't need anymore.

### CODE REVIEW

**Comment by jehna** [comment]
I'd rather use the jsdelivr CDN link diectly, so you can just copy-paste and start coding :)

File: README.md [hunk 1]
```diff
@@ -26,8 +26,9 @@ If you would like to contribute another port (which would be awesome!), please o
 ## How to get started
 ### In the browser
 ```HTML
-<script type="text/javascript" src="VerbalExpressions.js"></script>
+<script src="VerbalExpressions.js"></script>
 ```
+Or use the [jsDelivr CDN](http://www.jsdelivr.com/projects/jsverbalexpressions).
```
Comment (ionutvmi) [inline README.md]: please also add a code block that we can use to copy/paste the script on the page
`<script src="https://cdn.jsdelivr.net/jsverbalexpressions/0.2.0/verbalexpressions.min.js"></script>`

Comment (tomByrer) [inline README.md]: Or you could make an entire landing page with a [copy] button...
https://www.foundationcdn.com/

But is seems you guys know better than I what you want; perhaps make your
own PR?

Comment (ionutvmi) [inline README.md]: sure thing,
it will be added separately.

**Pull request merged**